frontend/packages/console-shared/src/utils/resource-utils.ts

```diff
@@ -114,6 +114,12 @@ export const getResourceList = (namespace: string, resList?: any) => {
       namespace,
       prop: 'statefulSets',
     },
+    {
+      isList: true,
+      kind: 'Secret',
+      namespace,
+      prop: 'secrets',
+    },
```

Comment:
@rohitkrai03 Do all helm secrets have some labels attached to them?

Should we add labels selectors? Or use the helm service instead?

Reply:
I've not seen any helm release secrets without the label till now. We could add a label selector for `owner=helm` but I think the idea of getting all the secrets and then matching them with the release list is safer.